src/login/components/LoginApp/ResetPassword/ResetPasswordMain.js

```diff
@@ -49,13 +51,23 @@ EmailForm = connect(() => ({
 }))(EmailForm);
 
 
-function ResetPasswordForm(props){
+function ResetPasswordMain(props){
   const dispatch = useDispatch();
+  const extra_security = useSelector(state => state.resetPassword.extra_security);
```

Comment:
I get extra_security, password_entropy, password_length, password_service_url, suggested_password in the payload from `GET_JSCONFIG_LOGIN_CONFIG_SUCCESS`. Should it be in both places or do I have old backend code?

Reply:
I can not see states(extra_security, password_entropy, password_length, password_service_url, suggested_password) in the payload from `GET_JSCONFIG_LOGIN_CONFIG_SUCCESS`. it should be in `POST_RESET_PASSWORD_VERIFY_EMAIL_SUCCESS`